### DISCUSSION TITLE
How do I display a table or relationship datalist in an action modal

### DISCUSSION THREAD

**Discussion by pslxx** [filamentphp/filament#8277] [General]
How do I display a table or relationship datalist in an action modal

**Comment by leandrocfe** [comment]
Could you please provide details about it?

**Comment by pslxx** [comment]
<img width="2226" alt="image" src="https://github.com/filamentphp/filament/assets/8417207/6a98b73e-32ba-4070-86a6-88d2b6e74d8d">
Like this!
Base table is a ORM Resource list table ,and the modal content is a relationship list table

How to use it quickly and achieve such an effect

**Reply by leandrocfe** [reply]
Not sure if it will work in a modal action. Maybe you can create a [custom page](https://filamentphp.com/docs/3.x/panels/resources/custom-pages) and [add a table to the livewire component](https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component)
